### DISCUSSION TITLE
Lenovo IdeaPad Pro 5 14AHP9 "  It Works It Works Ryzen 8845HS

### DISCUSSION THREAD

**Discussion by Blaubluter** [DavidS95/Smokeless_UMAF#84] [General]
Windows 11
IdeaPad Pro 5 14AHP9 Lenovo 
AMD 780M  (MAX 4 GB)
Ryzen 8845HS  (28GB DDR 7500 MHZ)

after

Windows 11
IdeaPad Pro 5 14AHP9 Lenovo 
AMD 780M  (MAX  8 GB)
Ryzen 8845HS  (24GB DDR 7500 MHZ)

I Love You 

 

 

The BIOS is broken due to the new setting. Can be started, but stops as soon as you go to the VRAM buffer settings. But it doesn't bother the device. But you can't set anything in the bios anymore, I don't care.

**Comment by juwonpee** [comment]
What?? So does it work or not?

**Comment by Blaubluter** [comment]
kanns nicht lesen ?

Am Fr., 17. Jan. 2025 um 23:17 Uhr schrieb Ju Won ***@***.***
